api/api.go

```diff
@@ -42,10 +42,13 @@ const (
 	SpecZones                = "zones"
 	SpecRacks                = "racks"
 	SpecRack                 = "rack"
-	SpecRegions              = "regions"
-	SpecLabels               = "labels"
-	SpecPriorityAlias        = "priority_io"
-	SpecIoProfile            = "io_profile"
+	// SpecBestEffort default is false. If set provisioning request will succeed
+	// even if specified data location parameters could not be satisfied.
+	SpecBestEffort    = "best_effort"
```

Comment:
nit: can we rename it to "best_effort_provisioning". This way it makes it clear to the user as to what this spec is doing. The term best_effort can be applied to many of the spec parameters here. For ex. if this spec does not apply to io_priority then we can name it specifically as "best_effort_location_provisioning" or something similar.